### PR TITLE
[REVIEW] DOC Adding Nx transition doc and preping for more

### PR DESCRIPTION
**Pull request by BradReesWork**
Updated docs conf.py file
added a PDF transition doc - migrating to HTML is next release

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: docs/Makefile [hunk 1]
```diff
@@ -7,6 +7,7 @@ SPHINXBUILD   = sphinx-build
 SPHINXPROJ    = cuGraph
 SOURCEDIR     = source
 BUILDDIR      = build
+PDFDIR		  = pdfs
```
Comment (afender) [inline docs/Makefile:10]: pdf has a number of issues:
- git cannot generate meaningful diffs, or merge binary files.
- growing repo and most importantly history size due to the binary nature of the format.

It's fine if we never update this pdf but we should consider porting this to rst or md if we think we'll update it.

File: docs/source/cugraph_intro.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* [Graph Theory](https://en.wikipedia.org/wiki/Graph_theory)
+* [Social Network Analysis](https://en.wikipedia.org/wiki/Social_network_analysis)
+
+cuGraph does not favor one field over another.  Our developers span the
+breadth of fields with the focus being to produce the best graph library
+possible.  However, each field has its own argot (jargon) for describing the
+graph (or network).  In our documentation, we try to be consistent.  In Python
+documentation we will mostly use the terms __Node__ and __Edge__ to better
+match NetworkX preferred term use, as well as other Python-based tools.  At
+the CUDA/C layer, we favor the mathematical terms of __Vertex__ and __Edge__.
```
Comment (afender) [inline docs/source/cugraph_intro.md:21]: I was hoping we could use the same terms everywhere in cugraph. I don't mind using `Node` in CUDA if that helps to be consistent. My only concern with Node is the conflict risk with "compute node".

File: docs/source/index.rst [hunk 3]
```diff
@@ -7,7 +7,7 @@ Welcome to cugraph's documentation!
 
    api.rst
    dask-cugraph.rst
-   cugraph_intro.rst
+   cugraph_intro.md
```
[the remark this answers is not in the document]
Comment (BradReesWork) [reply docs/source/index.rst:10]: yep.  I build and inspect locally.  Update the config to handle Markdown

File: docs/Makefile (hunk 1 above)
Comment (BradReesWork) [reply docs/Makefile:10]: yep.  But markdown is terrible on tables, images, formatting, etc..
I'll be replacing it in 0.17

File: docs/source/cugraph_intro.md (hunk 2 above)
Comment (BradReesWork) [reply docs/source/cugraph_intro.md:21]: Node / Edge is the preferred Nx, iGraph, etc. term use.

File: README.md [hunk 4]
```diff
@@ -57,16 +57,16 @@ for i in range(len(df_page)):
 | Layout       |                                        |              |                     |
 |              | Force Atlas 2                          | Single-GPU   |                     |
 | Link Analysis|                                        |              |                     |
-|              | Pagerank                               | Multiple-GPU | limited to 2 billion vertices |
-|              | Personal Pagerank                      | Multiple-GPU | limited to 2 billion vertices |
+|              | Pagerank                               | Multiple-GPU |                     |
+|              | Personal Pagerank                      | Multiple-GPU |                     |
```
[the remark this answers is not in the document]
Comment (BradReesWork) [reply README.md:61]: I'll fix this

File: docs/source/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (BradReesWork) [reply docs/source/index.rst]: switch back to RST

**Review by afender — APPROVED** [review]

**Comment by BradReesWork** [comment]
rerun tests

**Pull request merged**